### PR TITLE
chore: add redirects

### PR DESCRIPTION
**Pull request by gao-sun**
## Summary

add 301 redirects to prevent 404 issues on old urls for the new docs site

### CODE REVIEW

File: static/_redirects [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+/docs/references/tunnel-cli/ /customization/bring-your-ui/debug-and-test-your-custom-ui-locally 301
+/docs/references/tunnel-cli/deploy /customization/bring-your-ui/upload-custom-ui-assets-using-cli 301
+/docs/references/tunnel-cli/deploy/ /customization/bring-your-ui/upload-custom-ui-assets-using-cli 301
+/docs/references/users /user-management/user-data 301
+/docs/references/users/ /user-management/user-data 301
+/docs/references/users/custom-data /user-management/user-data#custom-data 301
+/docs/references/users/custom-data/ /user-management/user-data#custom-data 301
+/docs/references/users/social-identities /user-management/user-data#social-identities 301
+/docs/references/users/social-identities/ /user-management/user-data#social-identities 301
+/docs/references/using-cli/* /logto-oss/using-cli/:splat 301
```
Comment (charIeszhao) [inline static/_redirects:216]: Just curious what is `:splat`?

Comment (charIeszhao) [reply static/_redirects:216]: Aha, get it.

> On matching, a splat (asterisk, *) will greedily match all characters. You may only include a single splat in the URL.

Comment (gao-sun) [reply static/_redirects:216]: https://developers.cloudflare.com/pages/configuration/redirects/#splats

**Review by charIeszhao — APPROVED** [review]

File: static/_redirects (hunk 1 above)
Comment (charIeszhao) [reply static/_redirects:216]: yep, found it

**Pull request merged**